src/System.Data.DataSetExtensions/tests/System/Data/TypedTableBaseExtensionsTests.cs

```diff
@@ -1,39 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace System.Data.Tests
 {
     public class TypedTableBaseExtensionsTests
     {
         [Fact]
-        public void AsEnumerable_NullSource_ThrowsArgumentNullException()
+        public void Where_ThrowsArgumentNull()
```

Comment:
The tests in here look to be the same as what @hughbe added, just ordered differently. You should be able to revert this file.